include/CppUTestExt/MockCheckedExpectedCall.h

```diff
@@ -36,10 +36,12 @@ class MockCheckedExpectedCall : public MockExpectedCall
 
 public:
     MockCheckedExpectedCall();
+    MockCheckedExpectedCall(unsigned int numCalls);
     virtual ~MockCheckedExpectedCall();
 
     virtual MockExpectedCall& withName(const SimpleString& name) _override;
-    virtual MockExpectedCall& withCallOrder(unsigned int callOrder) _override;
+    virtual MockExpectedCall& withCallOrder(unsigned int callOrder) _override { return withCallOrder(callOrder, callOrder); }
```

Comment:
I'm not sure if this is the reason why @arstrube removed inline implementations, but it might be just because for testing inline methods can't be mocked up. 

In my projects I only accept inline implementations when these are only trivial calls to other overloaded methods for the same function (i.e. just adding default values or simple transformations of parameters). In these cases, it's fine doing it like that because it doesn't affect testability.